teamA/kimTaeksoo/model/userDao.js

```diff
@@ -1,40 +1,58 @@
 const myDataSource = require("../util/datasource");
 
 const createUser = async (name, email, profileImage, password) => {
-  await myDataSource.query(
-    `INSERT INTO users (
+  try {
+    await myDataSource.query(
+      `INSERT INTO users (
         name,
         email,
         profile_image,
         password
     ) VALUES (?,?,?,?)`,
-    [name, email, profileImage, password]
-  );
+      [name, email, profileImage, password]
+    );
+  } catch (err) {
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 500;
+    throw error;
+  }
 };
 
 const getUser = async () => {
-  const data = await myDataSource.query(
-    `SELECT
-        u.id,
-        u.email,
-        u.profile_image,
-        u.password
-    FROM users as u
-    `
-  );
-  return data;
+  try {
+    const data = await myDataSource.query(
+      `SELECT
+          u.id,
+          u.email,
+          u.profile_image,
+          u.password
+      FROM users as u
+      `
+    );
```

Comment:
SQL 쿼리문 포맷팅에 맞춰 수정해주세요 !
```suggestion
  try {
    const data = await myDataSource.query(
      `SELECT
          u.id,
          u.email,
          u.profile_image,
          u.password
      FROM users as u`
    );
```